### PR TITLE
Add utility methods to zebra-network.

### PR DESCRIPTION
**Pull request by hdevalence**
This adds two utility methods to zebra-network items:

- `AddressBook::potentially_connected_peers()`: the complement of the existing `AddressBook::disconnected_peers()`;
- `Network::default_port()`: gets the default port for a network type.

Extracted from #1008.

### CODE REVIEW

File: zebra-chain/src/parameters/network.rs [hunk 1]
```diff
@@ -11,6 +11,16 @@ pub enum Network {
     Testnet,
 }
 
+impl Network {
+    /// Get the default port associated to this network.
+    pub fn default_port(&self) -> u16 {
```
Comment (dconnolly) [inline zebra-chain/src/parameters/network.rs:16]: 👍

File: zebra-network/src/address_book.rs [hunk 2]
```diff
@@ -116,6 +116,18 @@ impl AddressBook {
         Utc::now() - CD::from_std(constants::LIVE_PEER_DURATION).unwrap()
     }
 
+    /// Used for range bounds, see cutoff_time
+    fn cutoff_meta() -> MetaAddr {
```
Comment (dconnolly) [inline zebra-network/src/address_book.rs:120]: 👍

File: zebra-network/src/address_book.rs [hunk 3]
```diff
@@ -137,18 +149,22 @@ impl AddressBook {
...
         self.by_time
-            .range((Excluded(cutoff_meta), Unbounded))
+            .range((Excluded(Self::cutoff_meta()), Unbounded))
+            .rev()
+            .cloned()
+    }
+
+    /// Return an iterator over peers that could potentially be connected, ordered from most
+    /// recently seen to least recently seen.
+    pub fn potentially_connected_peers<'a>(&'a self) -> impl Iterator<Item = MetaAddr> + 'a {
```
Comment (dconnolly) [inline zebra-network/src/address_book.rs:162]: 👍

**Review by dconnolly — APPROVED** [review]
Any tests? 🥰

**Review by yaahc — APPROVED** [review]

**Pull request merged**